src/util/saveArticles.ts

```diff
@@ -796,14 +800,20 @@ async function templateArticle(parsoidDoc: DominoElement, moduleDependencies: an
         styleDependenciesList: string[],
     };
 
+    if (webp) {
+        jsDependenciesList.push('webpHeroPolyfill');
+        jsDependenciesList.push('webpHeroBundle');
```

Comment:
Do we really need both? I doubt a bit that will work... but don't have tested.

Reply:
@kelson42 yes the function calling takes place polyfill.js but webpHeroBundle has something to do with conversions

Reply:
OK